Sources/FigmaExportCore/Processor/AssetsProcessor.swift

```diff
@@ -270,30 +299,40 @@ public extension AssetsProcessable {
             .filter { $0.platform == platform || $0.platform == nil }
             .sorted { $0.name < $1.name }
 
-        let darkColors = darkSet
-            .filter { $0.platform == platform || $0.platform == nil }
-            .sorted { $0.name < $1.name }
+        // After validations, only those dark colors in the light color palette are allowed
```

Comment:
This function works for any type of assets: colors, images. Let’s rename all “color” word to “asset” word.
lightColors → lightAssets, darkColorMap → darkAssetMap, etc.

Reply:
Good idea. Done.